cwl_tes/main.py

```diff
@@ -123,11 +123,15 @@ def main(args=None):
 
     if parsed_args.token:
         try:
+            validation_options = {}
+            validation_options['verify_aud'] = False
```

Comment:
This makes disabling audience verification permanent. Is there a way to make this behavior happen as a result of a command line flag?